Source/OmegaBank/Sources/Presentation/Map/Annotations/AtsMapAnnotation.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  AtsMapAnnotation.swift
+//  OmegaBank
+//
+//  Created by Konsantin Makhov on 14.07.2021.
+//  Copyright © 2021 RedMadRobot. All rights reserved.
+//
+
+import Foundation
+
+import MapKit
+import struct OmegaBankAPI.Atm
+
+final class AtmMapAnnotation: NSObject, MKAnnotation {
+    
+    let coordinate: CLLocationCoordinate2D
+    
+    let title: String?
+    
+    let subtitle: String?
+    
+    // MARK: - Init
+    
+    init(latitude: Double, longitude: Double, title: String?, subtitle: String?) {
+        self.coordinate = CLLocationCoordinate2D(latitude: latitude, longitude: longitude)
+        self.title = title
+        self.subtitle = subtitle
+    }
+}
+
+extension Atm {
+    var annotation: AtmMapAnnotation {
```

Comment:
Посколько мы приводим к протоколу `MKAnnotation` в сервисе

```suggestion
    var annotation: MKAnnotation {
```

Reply:
Поправил

Reply:
Если сделаем так, то мы не будем понимать что за массив к нам приходит и не сможем к нему подобрать соответствующую вьюшку

Reply:
Почему? Это просто протокол, возвращай также `AtmMapAnnotation`

Reply:
А нет, был не прав. Работает